mridata-3dfse-knee/templates/template.yaml

```diff
@@ -19,16 +27,20 @@ DATASETS:
 DATALOADER:
   DROP_LAST: True
   NUM_WORKERS: 8
+TIME_SCALE: "iter"  # Positive values are in iterations
 SOLVER:
   OPTIMIZER: "Adam"
   LR_SCHEDULER_NAME: ""
-  CHECKPOINT_PERIOD: -5000  # Checkpoint every 10 epochs
-  MAX_ITER: -8000  # Max number of epochs to train for
+  # Checkpointing frequency
+  CHECKPOINT_PERIOD: 5000
+  # Number of iterations to train for.
+  # We set this to 40000 for now. We may want to increase to 80000 later for a full run.
+  MAX_ITER: 40000
```

Comment:
Change this to 80000 - we'll run these experiments for more time for to all models to converge

Reply:
Will fix, thanks!